test/fast/Unit tests/nvm_default_packages

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+die () { echo "$@" ; cleanup ; exit 1; }
+cleanup () {
+  rm -rf "$(nvm_version_path v6.10.1)" ../../../default-packages
```

Comment:
tests are often run inside a working nvm installation; if I have a pre-existing `default-packages` file, this test will clobber it.